src/model/specific_path/Path.java

```diff
@@ -1,13 +1,67 @@
 package model.specific_path;
 
+import java.awt.Point;
+import java.util.ArrayList;
+
 import model.Shape;
 import model.ShapeName;
 
 public class Path extends Shape {
+	
```

Comment:
Yes, using Path. What I mean is, each PathPart is open. All together they might be closed, for example a traingle, but Pathpart cannot be directly a triangle